NEWS.md

```diff
@@ -1,5 +1,8 @@
 # testthat 0.10.0.9000
 
+* Added [Catch](https://github.com/philsquared/Catch) for unit testing of C++ code.
```

Comment:
Maybe we should leave NEWS commits until after merge? I've had to amend this many times now because of NEWS updated from various pull requests. :/